cookbooks/van-keken-vof/doc/van-keken-vof.tex

```diff
@@ -0,0 +1,216 @@
+    of the Van Keken problem made with the VOF interface tracking algorithm with five 
+    different global mesh refinements.
+    Since the VOF initial conditions are discontinuous, the above results should really be 
+    compared with the computations with discontinuous initial conditions on the left in 
+    Figure~\ref{fig:vk-2}. However, the above results also compare extremely favorably with the 
+    computations with smoothed, continuous initial conditions for the compositional field on 
+    the right in Figure~\ref{fig:vk-2}.
+    As in Figure~\ref{fig:vk-2}, 5 global refinements correspond to a $32 \times 32$ mesh and 10 global refinements correspond to a $1024 \times 1024$ mesh.}
+    \label{fig:vof-vk-1}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.4\textwidth]{cookbooks/van-keken-vof/doc/vof_van_keken_refinement_comparison.png}
+    \caption{\it The results of two computations of the Van Keken problem made with the VOF 
+        interface tracking algorithm overlaid upon each other at $t_{end}=2000$.
+        This visualization shows the reconstructed boundary between the two
+        materials at the final time $t_{end}$ as computed on a uniform grid
+        with 7 and 8 levels of refinement.
+        The boundaries between the materials are displayed as contours
+        of the fields $\tilde{\psi}^7\,(t_{end})$ (black) and
+        $\tilde{\psi}^8\,(t_{end})$ (bright green), which are generated by the  visualization postprocessor.
+        The contours for the reconstructed material boundaries are superimposed
+        on a color gradient visualization of the material composition for the
+        computation with 8 levels of refinement in order to make the regions
+        with each fluid type more evident.
+        Compare with the fourth image on the right in Figure~\ref{fig:vk-1}.
+    }
+    \label{fig:VOF_van_Keken-02}
+\end{figure}
+
+Since, as noted above, when the interface is discontinuous, the van Keken problem is a version 
+of the Rayleigh-Taylor problem, which is unstable to perturbations of all 
+wavelengths\footnote{This is true whether the two fluids have the same viscosity or different 
+viscosities.} (e.g. see~\cite{SC:1961}).
+Therefore, it is extremely sensitive to the initial conditions.
+In order to address this sensitivity, we do not use  the default approach of
+computing the initial material volume fractions using a composition quadrature.
+Instead we compute the initial volume fractions using a signed distance
+function $\phi$ as follows~\cite{JMR:2019,JMR-EGP:2019}.
+
+First we create the function $\phi$, which has the following two properties: 1) it is positive 
+in the region that contains one of the fluids, which we will refer to as fluid~1, and negative 
+in the complement of this region, which we will refer to as fluid~2, and 2) at each point in 
+the domain the magnitude $| \phi |$ of $\phi$ is the distance to the boundary between the 
```

Comment:
By definition, a signed distance function (which is the term for the type of function we define here) requires that it be the actual distance. In practice, the use of the horizontal or vertical distance is often close enough when considering small perturbations, but this is a case of a sufficiently close approximation rather than them actually satisfying the requirements for a signed distance function.

Reply:
Yes, I think below you make that distinction. Feel free to do with my comment whatever seems right to you.

Reply:
I added some text below for the purpose of trying to make the use clearer.